<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5A3OTB0" failures="2" tests="7" name="Test" time="8.326" errors="0" timestamp="2025-05-01T16:59:23 IST">
  <testcase classname="tests.DeleteUserTest" name="verifyDeleteUser" time="5.185"/>
  <testcase classname="tests.UpdateTest" name="verifyUpdatePatch" time="1.54"/>
  <testcase classname="tests.UpdateTest" name="verifyUpdatePut" time="0.532">
    <failure type="java.lang.AssertionError" message="failed status code for put request. expected [201] but found [200]">
      <![CDATA[java.lang.AssertionError: failed status code for put request. expected [201] but found [200]
at tests.UpdateTest.verifyUpdatePut(UpdateTest.java:20)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyUpdatePut -->
  <testcase classname="tests.UserTests" name="verifyCreateUser" time="0.521"/>
  <testcase classname="tests.UserTests" name="verifyUserNotFound" time="0.102">
    <failure type="java.lang.AssertionError" message="Status code failed. expected [403] but found [404]">
      <![CDATA[java.lang.AssertionError: Status code failed. expected [403] but found [404]
at tests.UserTests.verifyUserNotFound(UserTests.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserNotFound -->
  <testcase classname="tests.UserTests" name="verifygetUser" time="0.115"/>
  <testcase classname="tests.ListOfUsers" name="verifyGetAllUsers" time="0.123"/>
</testsuite> <!-- Test -->
